src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (numberOne) => {
+    const result = (numbertwo) => {
```

Comment:
According to cheklist 2, it will be better to make a direct return of the function.
```suggestion
    return (numbertwo) => {
```